app/src/main/java/com/zeapo/pwdstore/pwgenxkpwd/PasswordBuilder.kt

```diff
@@ -155,12 +136,10 @@ class PasswordBuilder(ctx: Context) {
         return password.toString()
     }
 
-    private fun capitalize(s: String): String {
-        var result = s
-        val lower = result.toLowerCase(Locale.getDefault())
-        result = lower.substring(0, 1).toUpperCase(Locale.getDefault()) + result.substring(1)
-        return result
-    }
+    private fun capitalize(s: String) =
```

Comment:
can this be replaced with the stdlib capitalize method?

Reply:
That call cares about even more Unicode case mapping weirdness, so I believe the answer is yes. I do love the Kotlin stdlib.